pkg/resource/processing_job/testdata/sdkapi/describe/success_completed_describe.json

```diff
@@ -0,0 +1,103 @@
+                "S3CompressionType": "None",
+                "S3DataDistributionType": "FullyReplicated",
+                "S3DataType": "S3Prefix",
+                "S3InputMode": "File",
+                "S3Uri": "s3://sagemaker-sample-data-us-west-2/algorithms/kmeans/mnist/mnist.pkl.gz"
+            }
+        },
+        {
+            "AppManaged": false,
+            "DatasetDefinition": null,
+            "InputName": "source_code",
+            "S3Input": {
+                "LocalPath": "/opt/ml/processing/code",
+                "S3CompressionType": "None",
+                "S3DataDistributionType": "FullyReplicated",
+                "S3DataType": "S3Prefix",
+                "S3InputMode": "File",
+                "S3Uri": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/processing/kmeans_preprocessing.py"
+            }
+        }
+    ],
+    "ProcessingJobArn": "arn:aws:sagemaker:us-west-2:123456789012:processing-job/unit-testing-processing-job",
+    "ProcessingJobName": "unit-testing-processing-job",
+    "ProcessingJobStatus": "Completed",
+    "ProcessingOutputConfig": {
+        "KmsKeyId": null,
+        "Outputs": [
+            {
+                "AppManaged": false,
+                "FeatureStoreOutput": null,
+                "OutputName": "train_data",
+                "S3Output": {
+                    "LocalPath": "/opt/ml/processing/output_train/",
+                    "S3UploadMode": "EndOfJob",
+                    "S3Uri": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/processing/output/"
+                }
+            },
+            {
+                "AppManaged": false,
+                "FeatureStoreOutput": null,
+                "OutputName": "test_data",
+                "S3Output": {
+                    "LocalPath": "/opt/ml/processing/output_test/",
+                    "S3UploadMode": "EndOfJob",
+                    "S3Uri": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/processing/output/"
+                }
+            },
+            {
+                "AppManaged": false,
+                "FeatureStoreOutput": null,
+                "OutputName": "valid_data",
+                "S3Output": {
+                    "LocalPath": "/opt/ml/processing/output_valid/",
+                    "S3UploadMode": "EndOfJob",
+                    "S3Uri": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/processing/output/"
+                }
+            }
+        ]
+    },
+    "ProcessingResources": {
+        "ClusterConfig": {
+            "InstanceCount": 1,
+            "InstanceType": "ml.m5.large",
+            "VolumeKmsKeyId": null,
+            "VolumeSizeInGB": 20
+        }
+    },
+    "ProcessingStartTime": null,
+    "RoleArn": "arn:aws:iam::123456789012:role/service-role/AmazonSageMaker-ExecutionRole-20210920T111639",
+    "StoppingCondition": {
+        "MaxRuntimeInSeconds": 86400
+    },
+    "TrainingJobArn": null
```

Comment:
Does this mean we arent testing that a successfully completed job has an ARN ? - can we add this as a comment in the test for future reference

Reply:
We are checking for the processing_job_arn, in this example the processingJob doesn't run trainingjobs so I believe trainingjobArn should be null.

Reply:
got it.